### PR TITLE
Arabic updates

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: channels/iq.m3u [hunk 1]
```diff
@@ -125,7 +125,5 @@ http://liveshell.net:1935/live/speda084k/playlist.m3u8
 https://602ccc850c9bb.streamlock.net/sterktv/smil:sterk.smil/playlist.m3u8
 #EXTINF:-1 tvg-id="Zagros.iq" tvg-name="Zagros" tvg-country="IQ" tvg-language="Arabic" tvg-logo="https://www.zagrosnews.net/sites/default/files/zagros-logo-01_1.png" group-title="General",Zagros (720p)
 https://5a3ed7a72ed4b.streamlock.net/zagrostv/SMIL:myStream.smil/playlist.m3u8
-#EXTINF:-1 tvg-id="ZagrosTV.iq" tvg-name="Zagros TV" tvg-country="IQ" tvg-language="" tvg-logo="" group-title="",Zagros TV (480p)
-http://163.172.39.215:25461/line/C4@!a3a1@!w72A/48
```
Comment (tilfizyawn) [inline channels/iq.m3u:129]: Duplicate of the line below

File: channels/jo.m3u [hunk 2]
```diff
@@ -7,9 +7,11 @@ https://ammantv.c.s73cdn.net/23153d43-375a-472a-bc5f-9827582b5d22/elemental/live
...
 #EXTINF:-1 tvg-id="JawharaFM.jo" tvg-name="Jawhara FM" tvg-country="JO" tvg-language="Arabic" tvg-logo="https://www.jawharafm.net/ar/static/fr/image/png/logo-jfm.png" group-title="Music",Jawhara FM (720p)
 http://streaming.toutech.net:1935/live/mp4:jawharafm.sdp/playlist.m3u8
-#EXTINF:-1 tvg-id="Jordan TV" tvg-name="Jordan TV" tvg-country="JO" tvg-language="Arabic" tvg-logo="https://i.imgur.com/siAiiws.jpg" group-title="General",Jordan TV
+#EXTINF:-1 tvg-id="Jordan TV" tvg-name="Jordan TV" tvg-country="JO" tvg-language="Arabic" tvg-logo="https://static.jrtv.gov.jo/resize-width/380/content/x/Mp/1/Poster.png" group-title="General",Jordan TV
 https://jrtv-live.ercdn.net/jordanhd/jordanhd.m3u8
-#EXTINF:-1 tvg-id="MelodyFM.jo" tvg-name="Melody FM" tvg-country="JO" tvg-language="Arabic" tvg-logo="http://melody-fm.com/img/Logo.png" group-title="Music",Melody FM (720p)
+#EXTINF:-1 tvg-id="Jordan TV" tvg-name="Jordan TV" tvg-country="JO" tvg-language="Arabic" tvg-logo="https://static.jrtv.gov.jo/resize-width/380/content/y/B5/2/Poster.png" group-title="Sports",Jordan Sport [Offline]
+https://jrtv-live.ercdn.net/jordansporthd/jordansporthd.m3u8
```
Comment (tilfizyawn) [inline channels/jo.m3u:13]: This might be Geo-blocked. I've marked it as Offline

File: channels/kw.m3u [hunk 3]
```diff
@@ -1,11 +1,9 @@
 #EXTM3U
-#EXTINF:-1 tvg-id="AlKassSports.kw" tvg-name="Al Kass Sports" tvg-country="KW" tvg-language="" tvg-logo="" group-title="Sports",Al Kass Sports (304p)
```
Comment (tilfizyawn) [inline channels/kw.m3u:2]: Alkass links are expired and don't work

File: channels/qa.m3u [hunk 4]
```diff
@@ -18,16 +18,6 @@ http://ott-cdn.ucom.am/s23/index.m3u8
 https://live-hls-web-aje.getaj.net/AJE/index.m3u8
 #EXTINF:-1 tvg-id="AlJazeeraMubasher.qa" tvg-name="Al Jazeera Mubasher" tvg-country="ARAB" tvg-language="Arabic" tvg-logo="https://i.imgur.com/xx6uOhZ.png" group-title="Legislative",Al Jazeera Mubasher (1080p)
 https://live-hls-web-ajm.getaj.net/AJM/index.m3u8
-#EXTINF:-1 tvg-id="AlKass.qa" tvg-name="Al Kass" tvg-country="QA" tvg-language="" tvg-logo="" group-title="Sports",Al Kass (304p)
```
Comment (tilfizyawn) [inline channels/qa.m3u:21]: Alkass links are expired and don't work

File: channels/qa.m3u [hunk 5]
```diff
@@ -42,7 +32,7 @@ https://www.livedoomovie.com/02_epl1_720p_UK/chunklist.m3u8
 #EXTINF:-1 tvg-id="beINSports2English.qa" tvg-name="beIN Sports 2 English" tvg-country="TH" tvg-language="English" tvg-logo="https://www.lyngsat.com/logo/tv/bb/bein-sports-english-2-qa.png" group-title="Sports",beIN Sports 2 English [Not 24/7]
 #EXTVLCOPT:http-user-agent=Mozilla/5.0 Macintosh; Intel Mac OS X 10_14_5 AppleWebKit/537.36 KHTML, like Gecko Chrome/76.0.3809.25 Safari/537.36
 https://www.livedoomovie.com/02_epl2_720p_UK/chunklist.m3u8
-#EXTINF:-1 tvg-id="BeINSportsXtra.qa" tvg-name="BeIN Sports Xtra" tvg-country="QA" tvg-language="" tvg-logo="" group-title="Sports",BeIN Sports Xtra (720p)
+#EXTINF:-1 tvg-id="BeINSportsXtra.qa" tvg-name="BeIN Sports Xtra" tvg-country="QA" tvg-language="English" tvg-logo="" group-title="Sports",BeIN Sports Xtra (720p)
```
Comment (tilfizyawn) [inline channels/qa.m3u:35]: Could not connect, but its an English channel

File: channels/jo.m3u (hunk 2 above)
Comment (Dum4G) [reply channels/jo.m3u:13]: 401 Auth required

**Review by Dum4G — APPROVED** [review]

**Pull request merged**